src/rewrite_clj/node/whitespace.clj

```diff
@@ -57,10 +71,18 @@
   "Create whitespace node."
   [s]
   {:pre [(string? s)
-         (re-matches #"(\s|,)+" s)
-         (not (re-matches #".*[\n\r].*" s))]}
+         (re-matches #"\s+" s)
+         (not (re-matches #".*[\n\r,].*" s))]}
   (->WhitespaceNode s))
 
+(defn comma-node
+  "Create comma node."
+  [s]
+  {:pre [(string? s)
+         (re-matches #",+" s)
+         (not (re-matches #".*[\n\r\s].*" s))]}
```

Comment:
I don't think the last precondition is necessary, since `re-matches` will (afaik) implicitly add `^...$` around the pattern (and something matching `^,+$` will never match the other regex).